Merging test to main

jwiebbelling opened the pull request.

Commit:
Botão de resetar e fonte adicionados

Commit:
Improvements UX

Added an if statement that prevents the user of not adding a value to the scores.

marcelorodrigo commented on canastra.js:
```diff
@@ -4,6 +4,12 @@ var totalTeamA = 0;
 
 function toSum() {
+  if (!elementA.value || !elementB.value) {
+    alert ("Please add a value!");
```
This message looks like it is in a different language used by the user interface.

marcelorodrigo replied:
Can we make an improvement and tell the user which amount it's invalid? It's team A amount or team B amount?

Commit:
Alerta atualizado

User recebe um alerta caso ambos os valores estiverem errados, casos individuais também são alertados (time A ou B)

marcelorodrigo reviewed (approved):
LGTM
